polynote-kernel/src/main/scala/polynote/kernel/PolyKernel.scala

```diff
@@ -41,33 +41,19 @@ class PolyKernel private[kernel] (
   private val launchingKernel = Semaphore[IO](1).unsafeRunSync()
   private val kernels = new ConcurrentHashMap[String, LanguageKernel[IO]]()
 
-
-  /**
-    * The symbol table, which stores all the currently existing runtime values
-    */
-  protected lazy val symbolTable = new RuntimeSymbolTable(globalInfo, statusUpdates)
-
   /**
     * The task queue, which tracks currently running and queued kernel tasks (i.e. cells to run)
     */
   protected lazy val taskQueue: TaskQueue = TaskQueue(statusUpdates).unsafeRunSync()
 
-  private def runPredef(kernel: LanguageKernel[IO], language: String): IO[Unit] =
-    kernel.predefCode.fold(kernel.init()) {
-      code => for {
-        oq <- Queue.unbounded[IO, Result]
-        _  <- taskQueue.runTaskIO(s"Predef $language", s"Predef ($language)")(_ => kernel.runCode("Predef", Nil, Nil, code, oq, statusUpdates))
-      } yield ()
-  }
-
   protected def getKernel(language: String): IO[LanguageKernel[IO]] = Option(kernels.get(language)).map(IO.pure).getOrElse {
     launchingKernel.acquire.bracket { _ =>
       Option(kernels.get(language)).map(IO.pure).getOrElse {
         for {
           factory <- IO.fromEither(Either.fromOption(subKernels.get(language), new RuntimeException(s"No kernel for language $language")))
-          kernel  <- taskQueue.runTask(s"Kernel$$$language", s"Starting $language kernel")(_ => factory.apply(dependencies.getOrElse(language, Nil), symbolTable))
+          kernelContext = new RuntimeContext(globalInfo)
```

Comment:
This is going to create a RuntimeContext for each language interpreter... is that intentional?

Reply:
Good point, no it's not intentional...